cli/connect/interactive.go

```diff
@@ -0,0 +1,372 @@
+	setFormatOption(newFormat, opts)
+}
+
+// handleFormatTTableShortOption handles ttable output format short interactive option.
+func handleFormatTTableShortOption(opts handlerOpts) {
+	if !validateFormatOption(opts) {
+		return
+	}
+
+	newFormat := strings.TrimPrefix(opts.trimmedSpace, setFormatShortPrefix)
+	setFormatOption(newFormat, opts)
+}
+
+// validateTableDialectOption validates table dialect and returns false if there is any problems.
+func validateTableDialectOption(opts handlerOpts) bool {
+	if opts.trimmedSpace == strings.TrimSpace(setTableDialectPrefix) {
+		log.Error("specify table dialect: default, markdown or jira")
+		return false
+	}
+	if opts.console.outputFormat != formatter.TableFormat &&
+		opts.console.outputFormat != formatter.TTableFormat {
+		log.Error("table dialects supports only" +
+			" for table and ttable output formats")
+		return false
+	}
+
+	return true
+}
+
+// handleTableDialectOption handles table dialect interactive option
+// for table/ttable output formats.
+func handleTableDialectOption(opts handlerOpts) {
+	if !validateTableDialectOption(opts) {
+		return
+	}
+
+	newTableDialect := opts.trimmedPrefix
+	if tableDialect, ok := formatter.ParseTableDialect(newTableDialect); ok {
+		opts.formatterOpts.TableDialect = tableDialect
+		if opts.formatterOpts.TableDialect != formatter.DefaultTableDialect &&
+			opts.formatterOpts.ColWidthMax > 0 {
+			opts.formatterOpts.ColWidthMax = 0
+			log.Info("selected max width: disabled")
+		}
+		log.Infof("selected table dialect: %s", opts.formatterOpts.TableDialect.String())
+	} else {
+		log.Warnf("unsupported table dialect: %s", newTableDialect)
+	}
+}
+
+// handlerOpts contains params for interactive option handler and its detector.
+type handlerOpts struct {
+	in            string
+	trimmedSpace  string
+	trimmedPrefix string
+	executorEval  *string
+	console       *Console
+	formatterOpts *formatter.Opts
+}
+
+// handleInteractiveOption handles slash options for interactive console and returns
+// true if slash options detected in user input.
+func handleInteractiveOption(in string, executorEval *string,
+	console *Console, formatterOpts *formatter.Opts) bool {
+
+	interactiveOptionHandlers := map[string]func(opts handlerOpts){
+		"\\help":                             handleHelpOption,
+		"help":                               handleHelpOption,
+		"\\":                                 handleHelpOption,
+		"\\?":                                handleHelpOption,
+		strings.TrimSpace(setLanguagePrefix): handleLanguageOption,
+		strings.TrimSpace(setTableColWidthLongPrefix):     handleWidthLongOption,
+		strings.TrimSpace(setTableColWidthShortPrefix):    handleWidthShortOption,
+		strings.TrimSpace(setPseudoGraphicsEnablePrefix):  handlePseudoGraphicsEnableOption,
+		strings.TrimSpace(setPseudoGraphicsDisablePrefix): handlePseudoGraphicsDisableOption,
+		strings.TrimSpace(setFormatLongPrefix):            handleFormatLongOption,
```

Comment:
There may be even fewer manual checks here:
```suggestion
		strings.TrimSpace(setFormatLongPrefix + " yaml"):            handleFormatYamlOption,
		//...
		strings.TrimSpace(setFormatLongPrefix):            handleFormatInvalidOption,
```